cfy_cluster_setup/main.py

```diff
@@ -392,15 +431,23 @@ def main():
         '--config-path',
         action='store',
         help='The completed cluster install configuration file. '
-             'default: ./cluster_config.yaml')
+             'default: ./{0}'.format(CLUSTER_CONFIG_FILE_NAME))
+
+    install_args.add_argument(
```

Comment:
As you're using this twice, you could farm out creation of it to a function, a la https://github.com/cloudify-cosmo/patchify/blob/master/patchify#L3499

Reply:
Thanks! I was looking for a way to do it.